docs-source/src/architecture.rst

```diff
@@ -2,7 +2,29 @@ License Manager Architecture
 ============================
 The License Manager is based on a client/backend architecture. The backend consists of a RESTful API built with Python over a
 PostgreSQL database. The client consists of a timed reconcile job that runs on the control node in the cluster and
-a prolog integration to Slurm.
+a prolog integration to Slurm. The License Manager may serve multiple Slurm clusters and account for license consumption
+originating outside these Slurm clusters.
+
+Licenses Manager concept
+------------------------
+The current Slurm (23.02) licence accounting capabilities are limited to handle only consumption occurring within the cluster
+using counters. Hence, license consumption originating in other clusters or workstations is not accounted for by this counter.
+The counter is setup defining the total number of licenses available and the number of consumed and free licenses are
+calculated based on current running jobs.
+
+When jobs are submitted to the cluster the number of licenses expected to be consumed is stated which allows for Slurm
+to only schedule jobs when licenses are available. When a job starts which requested licenses, the available licenses are
+reduced. If not sufficient licenses are available, the job is not deemed eligible for submission by Slurm's scheduler and
+will be held back in the queue.
+
+In a situation where license consumption may arise outside a single Slurm cluster, i.e. multiple clusters or consumed by
+other consumers, the internal counter of Slurm is not sufficient for efficient cluster and license utilization. The License
+Manager may then assist in keeping track of license consumption to continuously update the license availability to account
+for licenses consumed globally as well as licenses for jobs which have been started but not yet consumed the licenses from
+the license servers perspective.
```

Comment:
```suggestion
Manager provides improved license bookkeeping. It keeps track of license consumption to continuously update the license availability. To do this, License Manager accounts for global license consumption by interfacing directly with license servers. It also tracks licenses that have been reserved for jobs that have been started but not yet consumed the licenses from the license servers.
```

Reply:
This section could be further improved to get the nuances correct of the behavior. I will try to rework further.